### DISCUSSION TITLE
Can this project be compiled under linux？

### DISCUSSION THREAD

**Discussion by Mr-LZG** [PowerShell/Win32-OpenSSH#2068] [Q&A] [answered]
I need a cross-platform SSH library, can I use this project to compile on both windows and linux?

**Comment by maertendMSFT** [comment]
This project is currently focused on the Windows environment only.  Is there a reason why the inbox SSH for your linux distro is not sufficient?

**Reply by Mr-LZG** [reply, marked as answer]
I compiled successfully under linux.
